app/src/main/java/com/example/habitsmasher/DaysTracker.java

```diff
@@ -0,0 +1,261 @@
+package com.example.habitsmasher;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Set;
+
+public class DaysTracker {
+    /**
+     * This class is used to represent the days of the week a habit takes place, and contains all functionality in
+     * storing, querying, modifying, and returning what days of the week a habit takes place. This class does not currently handle
+     * storing the days in a database, but is capable of taking in information stored in a database and converting to a DaysTracker object.
+     *
+     * @author: Cameron Matthew
+     * @Version: 1.0
+     * @see Habit
+     */
+
+    //instance variables to represent days of the week
+    private ArrayList<Boolean> _days;
```

Comment:
Arraylist keeps track of the days, with Monday starting at 0 and Sunday at 6.

Reply:
Just asking about the reasoning behind the design choice here. So in the habit constructor, we are taking the days of the week as **two character strings**, but here they are booleans tied to the index (0-6) of this array.

Is there a reason why we can't have both implementations be consistent, to avoid confusion?

Reply:
I chose to use the mix of implementations for the ease of use of each implementation in their respective environments.

The string input is tied to keeping the fields of the DB clean; I believe it's easier to think about the days **outside of** the tracker as a string, and easier to think about the days **inside of** the tracker as an array.

With the internal representation being an array, I can use array methods, iterate smoothly over all elements, etc. 

Basically, its externally a string to be clear and concise, and internally an arraylist to be easy to use and less cluttered.